demos/SampleWPF/Converter/BooleanToVisibilityConverter.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Globalization;
+using System.Windows;
+using System.Windows.Data;
+
+namespace AiurVersionControl.SampleWPF.Converter
+{
+    /// <summary>
+    /// Converts a Boolean into a Visibility.
+    /// </summary>
+    public class BooleanToVisibilityConverter : IValueConverter
```

Comment:
I guess wpf already provided a tool like this. See how I display the server address bar.

Reply:
Wpf's 'BooleanToVisibilityConverter' seems complicate to convert 'False -> visible', so i add a isConvert parameter in it.